### PR TITLE
postgres: add new fields to meeting

### LINKED ISSUE #203: DB: new fields for meeting table
Connected to #111. Below you can find at least a subset of fields that need to be added to postgreSQL scheme.

According to these mocks https://xd.adobe.com/view/ca32af76-bb04-4e66-b0f4-0ddbb6525ea5-9f61/screen/473694e8-4243-41ca-b1c5-8a80fd4db6d1/, https://xd.adobe.com/view/ca32af76-bb04-4e66-b0f4-0ddbb6525ea5-9f61/screen/eb1d390c-462d-475a-af71-8d63bdb3b4fd/, https://xd.adobe.com/view/ca32af76-bb04-4e66-b0f4-0ddbb6525ea5-9f61/screen/0d931d9b-253f-4c73-9b0b-3cc89203e9f3/:
Screen Shot 2021-04-29 at 7 32 22 PM
Screen Shot 2021-04-29 at 7 46 55 PM
Screen Shot 2021-04-29 at 8 35 17 PM

we need more fields in the `meeting` table:

| UI field               | type           | 
|------------------------|----------------|
| Virtual Meeting ID     | String (possibly only digits?)        | 
| Call-in Information    | String (phone format)        |
| Email - Before Meeting | String (email format) |
| Email - During Meeting | String (email format) |
| eComment | String (url format) |
| City of San Jose link | String (url format) |
| YouTube link | String (url format) |

They should be editable only by admins.

### PR DESCRIPTION
**Pull request by sent-hil**
Closes #203

Also updates readme based on feedback (including changing the project name to match the new name).

### CODE REVIEW

**Comment by anna-kartynnik** [comment]
Could you please also add these new fields into 
https://github.com/codeforsanjose/gov-agenda-notifier/blob/f08a4fe482ae32f4685937634f23d30a6c785ef8/backend/graphql_api/lambda/graphql/apolloServer.js#L75-L84
Should I create a new issue for adding these fields into the corresponding mutations?

**Comment by sent-hil** [comment]
@anna-kartynnik no need! i updated the PR...do you want me to update https://github.com/codeforsanjose/gov-agenda-notifier/blob/new-fields/backend/graphql_api/lambda/graphql/apolloServer.js#L39 to include the new fields?

**Comment by anna-kartynnik** [comment]
Yes please, if you can update  https://github.com/codeforsanjose/gov-agenda-notifier/blob/new-fields/backend/graphql_api/lambda/graphql/apolloServer.js#L39,  https://github.com/codeforsanjose/gov-agenda-notifier/blob/new-fields/backend/graphql_api/lambda/graphql/apolloServer.js#L4 and their handlers https://github.com/codeforsanjose/gov-agenda-notifier/blob/develop/backend/graphql_api/lambda/graphql/resolvers/mutation.js#L39, https://github.com/codeforsanjose/gov-agenda-notifier/blob/develop/backend/graphql_api/lambda/graphql/resolvers/mutation.js#L180, that would be great!

**Review by JMStudiosJoe — APPROVED** [review]

**Comment by sent-hil** [comment]
changing to blocked, there were new requests to the story...will try to get to them this week

**Comment by sent-hil** [comment]
@anna-kartynnik @JMStudiosJoe this is ready again

**Review by JMStudiosJoe — DISMISSED** [review]
lgtm

File: backend/agenda_upload_service/lambda/db/dbClient.js [hunk 1]
```diff
@@ -125,24 +125,31 @@ module.exports = async (logger) => {
...
-  module.createMeeting = async (meetingType, meetingStartTimestamp, virtualMeetingUrl, status) => {
+  module.createMeeting = async (meetingType, meetingStartTimestamp, virtualMeetingUrl, status, virtualMeetingId, callInInformation, emailBeforeMeeting, emailDuringMeeting, eComment, cityOfSanJoseMeeting, youtubeLink) => {
     logger.info('dbClient: createMeeting');
     const now = Date.now();
     const createdTimestamp = now;
     const updatedTimestamp = now;
     const queryString = `
-        INSERT INTO meeting(meeting_type, meeting_start_timestamp, virtual_meeting_url, created_timestamp, updated_timestamp, status)
-        VALUES ($1, to_timestamp($2), $3, to_timestamp($4), to_timestamp($5), $6) 
+        INSERT INTO meeting(meeting_type, meeting_start_timestamp, virtual_meeting_url, created_timestamp, updated_timestamp, status, virtualMeetingId, callInInformation, emailBeforeMeeting, emailDuringMeeting, eComment, city_ofsanJoseMeeting, youtubeLink)
```
Comment (anna-kartynnik) [inline backend/agenda_upload_service/lambda/db/dbClient.js:134]: Shouldn't it be
```
INSERT INTO meeting(meeting_type, meeting_start_timestamp, virtual_meeting_url, created_timestamp, updated_timestamp, status, virtual_meeting_id, call_in_information, email_before_meeting, email_during_meeting, eComment, city_of_san_jose_meeting, youtube_link)
```
? Note the snake case for the column names.

File: backend/agenda_upload_service/lambda/db/dbClient.js [hunk 2]
```diff
@@ -125,24 +125,31 @@ module.exports = async (logger) => {
...
+        VALUES ($1, to_timestamp($2), $3, to_timestamp($4), to_timestamp($5), $6)
```
Comment (anna-kartynnik) [inline backend/agenda_upload_service/lambda/db/dbClient.js:135]: Forgot to add new values
```
VALUES ($1, to_timestamp($2), $3, to_timestamp($4), to_timestamp($5), $6, $7, $8, $9, $10, $11, $12, $13)
```
?

File: backend/graphql_api/lambda/graphql/apolloServer.js [hunk 3]
```diff
@@ -29,15 +29,15 @@ module.exports = (logger) => {
...
         # TODO: Need to add verify token query
-        
+
         # TODO: Need to add reset password query
     }
 
     type Mutation {
-        createMeeting(meeting_start_timestamp: String!, meeting_type: String, status: String, virtual_meeting_url: String): meeting
-        updateMeeting(id: Int!, status: String, meeting_type: String, virtual_meeting_url: String, meeting_start_timestamp: String, meeting_end_timestamp: String): meeting
+        createMeeting(meeting_start_timestamp: String!, meeting_type: String, status: String, virtual_meeting_url: String, virtual_meeting_id: String!, call_in_information: String, email_before_meeting: String, email_during_meeting: String, eComment: String, city_of_san_jose_meeting: String, youtube_link: String): meeting
```
Comment (anna-kartynnik) [inline backend/graphql_api/lambda/graphql/apolloServer.js:39]: It seems that a new meeting's date and time only should be required for creation (according to [these mocks](https://xd.adobe.com/view/ca32af76-bb04-4e66-b0f4-0ddbb6525ea5-9f61/screen/f446e0cf-d613-448d-9dac-1a0ec51f0b69/)). So I think `virtual_meeting_id` shouldn't be required.

**Review by anna-kartynnik — CHANGES_REQUESTED** [review]
Looks good but has minor issues, see my comments.

**Comment by anna-kartynnik** [comment]
@sent-hil if you don't mind I'd like to make the necessary changes to finish this PR.

**Comment by sent-hil** [comment]
@anna-kartynnik that would be great, thank you

**Review by anna-kartynnik — APPROVED** [review]

**Comment by anna-kartynnik** [comment]
@JMStudiosJoe I've fixed the comments above, so it's ready to be merged.

**Comment by sent-hil** [comment]
Thank you @anna-kartynnik

**Pull request merged**